classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3599,6 +3638,7 @@ public static function webhook_is_working() {
 			SELECT COUNT(*)
 			FROM $wpdb->pmpro_membership_orders
 			WHERE gateway = 'stripe'
+			AND gateway_environment = '" . $gateway_environment . "'
```

Comment:
This and the other vars should be passed in via `%s` and a `$wpdb->prepare()` call

Reply:
Fixed in commit https://github.com/strangerstudios/paid-memberships-pro/pull/1682/commits/d2055a05213a0f8e434a783f5a388b39921fa2ea